lib/views/index.erb

```diff
@@ -1,30 +1,56 @@
 <% if @my_cart.count != 0 %>
-  <div id="shopping_cart">
-    Shopping cart
+<div class="cell">
```

Comment:
Also, I can't see where `@my_cart` is created? What am I missing? Do you have a model for Cart?